main.go

```diff
@@ -204,12 +211,34 @@ func runCopy(db *redshift.Redshift, inputConf s3filepath.S3File, inputTable reds
 	return nil
 }
 
+func startEndFromGranularity(t time.Time, granularity string) (time.Time, time.Time) {
+	var duration time.Duration
+	if granularity == "day" {
+		duration = time.Hour * 24
+	} else {
+		duration = time.Hour * 24 * 7
```

Comment:
perhaps we should hardcode this for `granularity == "hour"` in case we add other granularities in the future? Otherwise it will always default to this